addons/google_account/models/google_service.py

```diff
@@ -20,142 +18,78 @@
 GOOGLE_API_BASE_URL = 'https://www.googleapis.com'
 
 
-class GoogleService(models.AbstractModel):
-    _name = 'google.service'
-    _description = 'Google Service'
+def _get_client_secret(ICP_sudo, service):
```

Comment:
do we really need an helper for this? especially given the fact it returns a secret

Reply:
Yes, it is all the purpose of this PR, make it overridable to avoid to duplicate all codes called by each method that uses this secret key.